os/net/link-stats.c

```diff
@@ -98,30 +110,26 @@ link_stats_is_fresh(const struct link_stats *stats)
 }
 /*---------------------------------------------------------------------------*/
 #if LINK_STATS_INIT_ETX_FROM_RSSI
+/*
+ * Returns initial ETX value from an RSSI value.
+ *    RSSI >= RSSI_HIGH           -> use default ETX
+ *    RSSI_LOW < RSSI < RSSI_HIGH -> ETX is a linear function of RSSI
+ *    RSSI <= RSSI_LOW            -> use minimal initial ETX
+ **/
 uint16_t
```

Comment:
This function could be `static`, couldn't it?

Reply:
changed and pushed